pkg/referrerstore/oras/authprovider/authprovider.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright The Ratify Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package authprovider
+
+import (
```

Comment:
if it is the default auth provider, do we need to name the file accordingly?

Reply:
I can rename it. It contains the AuthProvider Interface definition too. Is that ok? or should we separate them into separate files?